django/contrib/auth/password_validation.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import unicode_literals
+
+import os
+import re
+from difflib import SequenceMatcher
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured, ValidationError
+from django.template.defaultfilters import unordered_list
+from django.utils.encoding import force_text
+from django.utils.module_loading import import_string
+from django.utils.six import string_types
+from django.utils.translation import ugettext as _
+
+
+def get_password_validators():
+    validators = []
+    for validator in settings.AUTH_PASSWORD_VALIDATORS:
+        try:
+            klass = import_string(validator['NAME'])
+            validators.append(klass(**validator.get('OPTIONS', {})))
+        except ImportError:
+            raise ImproperlyConfigured("Invalid NAME for a password validator: %s. Check "
+                                       "your AUTH_PASSWORD_VALIDATORS setting." % validator['NAME'])
+
+    return validators
+
+
+def validate_password(password, user=None):
+    """
+    Validates whether the password meets all validator requirements.
```

Comment:
we're now using pep8 style for docstrings "Validate whether ..."   "return None", "raise ValidationError", etc.